src/hotspot/share/runtime/os.cpp

```diff
@@ -1669,8 +1669,11 @@ char* os::attempt_reserve_memory_at(char* addr, size_t bytes) {
   if (result != NULL) {
     MemTracker::record_virtual_memory_reserve((address)result, bytes, CALLER_PC);
   } else {
-    log_info(metaspace)("Attempt to reserve memory at " INTPTR_FORMAT " for "
-                         SIZE_FORMAT " bytes failed", p2i(addr), bytes);
+    LogTarget(Debug, os, metaspace) lt;
```

Comment:
You only needed to change:
`log_info(metaspace)("...");`
to
`log_debug(os, metaspace)("...")`

Reply:
I'd opt for info. This is a hard bug, which - almost - is worth a log_warning. It should not get drowned in debug or trace.